frontend/src/element/signup.ts

```diff
@@ -693,16 +700,17 @@ export class Signup extends LitElement {
         <h2>Where do you work?</h2>
         <label>
           <span class="title"
-            >Employement type${optionalLabel('employment-type')}</span
+            >Employement type${this.optionalLabel('employment-type')}</span
```

Comment:
While we're here, let's fix a typo in the existing text:

```suggestion
            >Employment type${this.optionalLabel('employment-type')}</span
```